
26/04/2022 11:59:30
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2656, in get_invoice_details\n    edit_data = get("http://"+self.ip+":5000/sales/edit" , params = {\'billNo\' : self.sale_id ,\'dbYear\':self.year , \'user\' : self.user  , \'sale_date\' : cust[0][\'sale_date\'] , \'cust_name\' : cust[0][\'acc_name\']})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 12:00:32
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2656, in get_invoice_details\n    edit_data = get("http://"+self.ip+":5000/sales/edit" , params = {\'billNo\' : self.sale_id ,\'dbYear\':self.year , \'user\' : self.user  , \'sale_date\' : cust[0][\'sale_date\'] , \'cust_name\' : cust[0][\'acc_name\']})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 12:01:16
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2656, in get_invoice_details\n    edit_data = get("http://"+self.ip+":5000/sales/edit" , params = {\'billNo\' : self.sale_id ,\'dbYear\':self.year , \'user\' : self.user  , \'sale_date\' : cust[0][\'sale_date\'] , \'cust_name\' : cust[0][\'acc_name\']})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 12:04:48
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2656, in get_invoice_details\n    edit_data = get("http://"+self.ip+":5000/sales/edit" , params = {\'billNo\' : self.sale_id ,\'dbYear\':self.year , \'user\' : self.user  , \'sale_date\' : cust[0][\'sale_date\'] , \'cust_name\' : cust[0][\'acc_name\']})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 12:05:45
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 872, in <lambda>\n    self.btn_save = ttk.Button(self.btn_frame , text = "Save" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.save(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2129, in save\n    pdf = get("http://"+self.ip+":7000/sales/invoice" , params = {\'billNo\' : saved_bill_no, \'Date\' : bill_date, \'customerName\': self.ent_cust_name.get().title() ,\'invoiceData\':self.billInfo,\'billTotal\': grand_total, \'oldBal\' : oldBal, \'oldBalData\':self.voucher,\'page\':self.rad_printer.get()})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 12:32:11
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 12:32:17
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 12:32:55
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2225, in cancel\n    get("http://"+self.ip+":5000/sales/cancelSales" , params = {"sale_id" : self.sale_id})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 12:34:30
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 12:35:24
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 12:36:52
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2225, in cancel\n    get("http://"+self.ip+":5000/sales/cancelSales" , params = {"sale_id" : self.sale_id})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 13:00:57
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1406, in enter_to_treeview\n    for each in self.prod_units[1:-1].split(","):\n']

26/04/2022 13:03:00
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1406, in enter_to_treeview\n    for each in self.prod_units.split(","):\n']

26/04/2022 13:04:14
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1406, in enter_to_treeview\n    print(self.prod_units.split(","))\n']

26/04/2022 13:04:16
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1406, in enter_to_treeview\n    print(self.prod_units.split(","))\n']

26/04/2022 13:16:09
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 15:25:43
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 15:40:56
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\models.py", line 898, in json\n    return complexjson.loads(self.text, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\__init__.py", line 357, in loads\n    return _default_decoder.decode(s)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 355, in raw_decode\n    raise JSONDecodeError("Expecting value", s, err.value) from None\n']

26/04/2022 15:42:23
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\models.py", line 898, in json\n    return complexjson.loads(self.text, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\__init__.py", line 357, in loads\n    return _default_decoder.decode(s)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 355, in raw_decode\n    raise JSONDecodeError("Expecting value", s, err.value) from None\n']

26/04/2022 15:43:34
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2240, in cancel\n    get("http://"+self.ip+":5000/sales/cancelSales" , params = {"sale_id" : self.sale_id})\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 15:44:18
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\models.py", line 898, in json\n    return complexjson.loads(self.text, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\__init__.py", line 357, in loads\n    return _default_decoder.decode(s)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 337, in decode\n    obj, end = self.raw_decode(s, idx=_w(s, 0).end())\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\json\\decoder.py", line 355, in raw_decode\n    raise JSONDecodeError("Expecting value", s, err.value) from None\n']

26/04/2022 15:48:28
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 15:57:49
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 1011, in get_cust_details\n    req = get("http://"+self.ip+":5000/sales/addEditNewSalesDetails" , params = params).json()\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']

26/04/2022 16:36:03
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2236, in cancel\n    get("http://"+self.ip+":5000/sales/removeSalesProduct" , params = {"product" : values , \'sale_id\' : self.sale_id ,  \'newBill\' : \'True\' , \'db_year\' : self.year} )\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 16:36:14
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2236, in cancel\n    get("http://"+self.ip+":5000/sales/removeSalesProduct" , params = {"product" : values , \'sale_id\' : self.sale_id ,  \'newBill\' : \'True\' , \'db_year\' : self.year} )\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 516, in send\n    raise ConnectionError(e, request=request)\n']

26/04/2022 16:38:55
['  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\tkinter\\__init__.py", line 1883, in __call__\n    return self.func(*args)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 874, in <lambda>\n    self.btn_cancel = ttk.Button(self.btn_frame , text = "Cancel" , width = 6 , style = "window_btn_medium.TButton" ,command = lambda : self.cancel(None) , state = con.DISABLED)\n', '  File "c:\\Users\\vijay\\Desktop\\Hosangadi2.0\\frontend\\sales.py", line 2236, in cancel\n    get("http://"+self.ip+":5000/sales/removeSalesProduct" , params = {"product" : values , \'sale_id\' : self.sale_id ,  \'newBill\' : \'True\' , \'db_year\' : self.year} )\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 76, in get\n    return request(\'get\', url, params=params, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\api.py", line 61, in request\n    return session.request(method=method, url=url, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 530, in request\n    resp = self.send(prep, **send_kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\sessions.py", line 643, in send\n    r = adapter.send(request, **kwargs)\n', '  File "C:\\Users\\vijay\\AppData\\Local\\Programs\\Python\\Python38\\lib\\site-packages\\requests\\adapters.py", line 498, in send\n    raise ConnectionError(err, request=request)\n']
